x-pack/metricbeat/module/aws/cloudwatch/metadata/rds/rds.go

```diff
@@ -31,8 +30,7 @@ func AddMetadata(regionName string, awsConfig awssdk.Config, fips_enabled bool,
 	// Get DBInstance IDs per region
 	dbDetailsMap, err := getDBInstancesPerRegion(svc)
 	if err != nil {
-		logp.Error(fmt.Errorf("getInstancesPerRegion failed, skipping region %s: %w", regionName, err))
-		return events, nil
+		return events, fmt.Errorf("getInstancesPerRegion failed, skipping region %s: %w", regionName, err)
```

Comment:
The "getInstancesPerRegion failed" is helpful for developers but saying that "the `aws.rds.db_instance` fields" are not available because of an error helps users in their troubleshooting journey.

Reply:
Great idea. As I understand it, I should also set up EC2/SQS just to see exactly what fields are missing, correct?

Reply:
Yeah, it would be an excellent hint for the troubleshooters. 

We can look for the exact field names using `metadataPrefix`:

https://github.com/elastic/beats/blob/9e83729ced98e391ac53c9ab3854cd26d3771e7d/x-pack/metricbeat/module/aws/cloudwatch/metadata/rds/rds.go#L21

https://github.com/elastic/beats/blob/9e83729ced98e391ac53c9ab3854cd26d3771e7d/x-pack/metricbeat/module/aws/cloudwatch/metadata/sqs/sqs.go#L19

https://github.com/elastic/beats/blob/9e83729ced98e391ac53c9ab3854cd26d3771e7d/x-pack/metricbeat/module/aws/cloudwatch/metadata/ec2/ec2.go#L20